packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -63,7 +67,9 @@ test.describe('Call Composite E2E Configuration Screen Tests', () => {
       return !!videoNode && videoNode.readyState === 4 && !videoNode.paused && videoNode;
     });
     await stubLocalCameraName(page);
+    await disableTooltips(page);
```

Comment:
Should we add a screenshot helper instead? `page.screenshotWithStubbedElements()` that internally disables tooltips and stubs out local camera name while taking the screenshot?

Reply:
Good idea.
However, might end up being used for scenarios where we need tooltips to be shown as well.
Someone new in the team might be unaware of what's happening behind the scenes and could end up wasting a lot of time debugging...

Reply:
Either way is fine, as long as we modify this PR to keep one way of avoiding tooltips instead of two.